migrations/Version20240130173300.php

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
ah so there were actually fields that we (foreign key) ids but did not have a _id postfix ? nice to fix this